politeiawww/cmsdatabase/cockroachdb/models.go

```diff
@@ -24,6 +24,11 @@ type Invoice struct {
 	UserSignature      string    `gorm:"not null"`
 	ServerSignature    string    `gorm:"not null"`
 	Version            string    `gorm:"not null"`
```

Comment:
Move version to be the first item of struct to remain consistent with rest of code.